x11-calc.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.mike632t.x11_calc</id>
+
+  <name>RPN Calculator</name>
+  <summary>An HP calculator emulator</summary>
```

Comment:
```suggestion
  <summary>HP calculator emulator</summary>
```

Reply:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#dont-start-with-an-article